docker-compose.yml.twig

```diff
@@ -111,6 +111,8 @@ services:
       - MYSQL_DATABASE=${DB_NAME}
       - MYSQL_USER=${DB_USER}
       - MYSQL_PASSWORD=${DB_PASS}
+    ports:
+      - "3306:3306"
```

Comment:
Having it just as `- 3306` would let docker pick a random port, though it will change each time the project is brought up

Reply:
Ah nice. What do you think to modifying the Welcome message to include the database connection details?

https://github.com/inviqa/harness-magento1/blob/c9f86f7f89a3b8c33a2aa8ac89b3179a8a285f8e/docker/image/console/root/lib/task/welcome.sh#L3

Probably not the ideal solution but developers can access the `mysql` container through a GUI tool without modifying harness config each time.

Reply:
as the welcome message could quickly get lost ive tweaked the PR with a possible approach, eg.

`
➜  blog: ws port mysql                                              
3306/tcp -> 0.0.0.0:32773
`

Reply:
I presume that this is a common use case and it may be a pain to reconfigure GUI tools each time the project starts. Perhaps a global service like traefik can be added?
Traefik can't handle TCP streams just yet unfortunately, but that doesn't mean another global workspace service couldn't bind to port 3306 and forward requests. NGINX can do TCP forwarding but the dynamic configuration would be a problem.